soc/silabs/silabs_s2/efr32mg24/Kconfig.soc

```diff
@@ -19,15 +19,21 @@ config SOC_PART_NUMBER_EFR32MG24B310F1536IM48
 	bool
 	select SOC_SERIES_EFR32MG24
 
+config SOC_PART_NUMBER_EFR32MG24B210F1536IM48
+	bool
+	select SOC_SERIES_EFR32MG24
+
```

Comment:
needs splitting into 2 or 3 commits, one adding the soc/dts, the other adding the board

Reply:
Done